src/main.js

```diff
@@ -488,6 +503,7 @@ class MiniMediaPlayer extends LitElement {
           @change=${e => this.player.setVolume(e)}
           @click=${e => e.stopPropagation()}
           min='0' max=${this.config.max_volume} value=${this.player.vol * 100}
+          dir=${this._computeRTLDirection(this.hass)}
```

Comment:
If we take the member variable approach, we could do 
```js
?rtl=${this.rtl}
```
here. Would have to change the slider patch slightly for the selector to work.